src/components/Transitions/ComponentTransition.tsx

```diff
@@ -0,0 +1,19 @@
+import { AnimatePresence, motion } from 'framer-motion';
+import { ReactElement } from 'react';
+
+type Props = {
+  children: React.ReactNode;
+  lineItemsExist: boolean;
+};
+
+export const ComponentTransition = (props: Props): ReactElement => {
+  return (
+    <AnimatePresence>
+      {!props.lineItemsExist && (
+        <motion.div key="1" initial={{ opacity: 0, scale: 1 }} animate={{ opacity: 1, scale: 1 }} transition={{ duration: 1 }} exit={{ opacity: 0 }}>
```

Comment:
Minor nitpick but feels like the `key` prop should be a number not a string?

Reply:
actually does this need a key at all if it isn't being mapped?

Reply:
Yeah good spot, I've just removed it